velox/connectors/hive/HiveWriteProtocol.h

```diff
@@ -31,6 +31,8 @@ class HiveWriterParameters : public WriterParameters {
 
   /// @param updateMode Write the files to a new directory, or append to an
   /// existing directory or overwrite an existing directory.
+  /// @param partitionName The name of partition if any that the writer writes
```

Comment:
Would you expand the documentation to explain what partitionName is and how it is used in combination with targetDirectory and writeDirectory?

My understanding is partitionName is actually a path to partition subdirectory that looks like a=<a>/b=<b>/c=<c>, where a, b and c are partition keys. It is appended to targetDirectory and writeDirectory paths to create the actual path to write to.

Reply:
Yes, exactly. partitionName is actually the partition subdirectory in string, in the format of a=/b=/c=. Revising the documentation.